assets/batch_score/components/driver/batch_score_llm/spec.yaml

```diff
@@ -2,7 +2,7 @@ $schema: http://azureml/sdk-2-0/ParallelComponent.json
 type: parallel
 
 name: batch_score_llm
-version: 1.0.3
+version: 1.0.4
```

Comment:
In one of the telemetry PR I saw that we were using it to log the component version. Can you update the constants file as well?

Reply:
Oh nvm - that has changed